app/actors/Actors.scala

```diff
@@ -1,47 +1,46 @@
 package actors
 
+import javax.inject._
+
+import akka.actor.{ActorRef, ActorSystem}
+import com.google.inject.AbstractModule
 import play.api._
-import play.api.libs.concurrent.Akka
+import play.api.libs.concurrent.AkkaGuiceSupport
 import backend._
 import akka.cluster.Cluster
 import java.net.URL
 
 /**
- * Lookup for actors used by the web front end.
+ * Guice module that provides actors.
+ *
+ * Registered in application.conf.
  */
-object Actors {
-
-  private def actors(implicit app: Application) = app.plugin[Actors]
-    .getOrElse(sys.error("Actors plugin not registered"))
-
-  /**
-   * Get the region manager client.
-   */
-  def regionManagerClient(implicit app: Application) = actors.regionManagerClient
+class Actors extends AbstractModule with AkkaGuiceSupport {
+
+  def configure() = {
+    // Bind the region manager client
+    bindActor[RegionManagerClient]("regionManagerClient")
```

Comment:
It does a few things. First, it ensures one, and only one of the actor is started at start up. Second, it allows the actor to be dependency injected with Play components on start up. Third, it allows the actors ActorRef to be injected into Play components (eg controllers), using `@Named("actor name")`. If you're embracing Guice in the rest of your app, it's a really nice abstraction to use, it fits in really nicely with the rest of Guice.